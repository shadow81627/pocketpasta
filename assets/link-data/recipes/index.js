// import recipe from './spaghetti.json';
// import pastaAglioEOlio from './pasta_aglio_e_olio.json';
// import lasagnaSalad from './lasagna-salad.json';
// import roastTaters from './roast-taters.json';
// import recipe5 from './vegan_hot_and_sour_soup_with_bok_choy.json';
// import trashCanNachos from './trash_can_nachos.json';
// import trashCanNachos2 from './trash_can_nachos_2.json';

const recipe = require('./spaghetti.json');
const pastaAglioEOlio = require('./pasta_aglio_e_olio.json');
const lasagnaSalad = require('./lasagna-salad.json');
const roastTaters = require('./roast-taters.json');
const recipe5 = require('./vegan_hot_and_sour_soup_with_bok_choy.json');
const trashCanNachos = require('./trash_can_nachos.json');
const trashCanNachos2 = require('./trash_can_nachos_2.json');

module.exports = [
  recipe,
  pastaAglioEOlio,
  lasagnaSalad,
  roastTaters,
  recipe5,
  trashCanNachos,
  trashCanNachos2,
];
