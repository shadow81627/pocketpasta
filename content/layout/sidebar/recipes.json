{
  "icon": "mdi:cook",
  "name": "Recipes",
  "route": "recipes",
  "pos": "1/1"
}
