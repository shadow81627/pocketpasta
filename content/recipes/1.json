{
  "@type": "Recipe",
  "aggregateRating": {
    "@type": "AggregateRating",
    "bestRating": "100",
    "ratingCount": "217",
    "ratingValue": "92",
    "worstRating": "0"
  },
  "createdAt": "2020-10-08T05:15:28.186Z",
  "datePublished": "2018-06-08T23:54:54",
  "description": "Here's what you need: extra virgin olive oil, small onion, celery, garlic, fresh basil, canned whole peeled tomato, salt, dried spathetti, freshly grated parmigiano-reggiano cheese, carrots",
  "image": "https://img.buzzfeed.com/thumbnailer-prod-us-east-1/video-api/assets/162208.jpg",
  "keywords": "audrey, audrey hepburn, devin lytle, hepburn, history, history bites, jared nunn, pasta, pomodoro, sauce, spaghetti",
  "name": "Audrey Hepburn’s Spaghetti Al Pomodoro Recipe by Tasty",
  "nutrition": {
    "@type": "NutritionInformation",
    "calories": "585 calories",
    "carbohydrateContent": "98 grams",
    "fatContent": "12 grams",
    "fiberContent": "9 grams",
    "proteinContent": "21 grams",
    "sugarContent": "11 grams"
  },
  "publisher": {
    "@type": "Organization",
    "logo": {
      "@type": "ImageObject",
      "url": "https://tasty.co/favicon.ico"
    },
    "name": "Tasty",
    "url": "https://pocketpasta.com"
  },
  "recipeCategory": "Dinner",
  "recipeCuisine": "Italian",
  "recipeIngredient": [
    "1 small onion, diced",
    "2 cloves garlic, minced",
    "2 carrots, diced small",
    "2 stalks celery, diced small",
    "2 large cans (28 oz. each) peeled Italian Roma tomatoes",
    "1 large bunch fresh basil, washed",
    "2 tbsp extra virgin olive oil",
    "1 lb. spaghetti pasta",
    "Salt to taste"
  ],
  "recipeInstructions": [
    {
      "@type": "HowToStep",
      "text": "Dice the carrots, onion, and celery. Thinly slice half of the basil leaves. Set aside."
    },
    {
      "@type": "HowToStep",
      "text": "Heat the olive oil in a large pot over medium heat. Add the onion, carrots, celery, and garlic. Cook until softened but not brown."
    },
    {
      "@type": "HowToStep",
      "text": "Add in the whole basil leaves, garlic, and tomatoes. Bring the sauce to a simmer, cover, and cook for 45 minutes, stirring occasionally and breaking apart the larger tomatoes as they cook."
    },
    {
      "@type": "HowToStep",
      "text": "After 45 minutes, or when the vegetables are tender, remove the sauce from the heat and let rest for 15 minutes."
    },
    {
      "@type": "HowToStep",
      "text": "While the sauce is simmering, fill another large pot with water and bring to a boil. Add salt, if desired. Cook the spaghetti until al dente. Drain the pasta."
    },
    {
      "@type": "HowToStep",
      "text": "Taste the sauce and add salt, if desired. Serve pasta topped generously with sauce, and the reserved sliced basil leaves."
    },
    {
      "@type": "HowToStep",
      "text": "While the sauce is simmering, fill another large pot with water and bring to a boil. Add salt, if desired. Cook the spaghetti until al dente. Drain and rinse the pasta with lukewarm water to prevent sticking."
    },
    {
      "@type": "HowToStep",
      "text": "Taste the sauce and add salt, if desired. Serve pasta topped generously with sauce, grated Parmigiano-Reggiano cheese, and the reserved sliced basil leaves."
    },
    {
      "@type": "HowToStep",
      "text": "Enjoy."
    }
  ],
  "recipeYield": "4 servings",
  "sameAs": [
    "https://tasty.co/recipe/audrey-hepburns-spaghetti-al-pomodoro",
    "https://www.pbs.org/food/recipes/audrey-hepburn-spaghetti-al-pomodoro"
  ],
  "suitableForDiet": "https://schema.org/VeganDiet",
  "tool": "Tongs",
  "updatedAt": "2020-10-08T05:15:28.186Z",
  "video": {
    "@type": "VideoObject",
    "contentUrl": "https://vid.tasty.co/output/97978/landscape_480/1528506768",
    "description": "Here's what you need: extra virgin olive oil, small onion, celery, garlic, fresh basil, canned whole peeled tomato, salt, dried spathetti, freshly grated parmigiano-reggiano cheese, carrots",
    "duration": "PT15M37S",
    "name": "Audrey Hepburn’s Spaghetti Al Pomodoro Recipe by Tasty",
    "publisher": {
      "@type": "Organization",
      "logo": {
        "@type": "ImageObject",
        "url": "https://tasty.co/favicon.ico"
      },
      "name": "Tasty"
    },
    "thumbnailUrl": "https://img.buzzfeed.com/thumbnailer-prod-us-east-1/video-api/assets/162208.jpg",
    "uploadDate": "2018-06-08T23:54:54"
  }
}
