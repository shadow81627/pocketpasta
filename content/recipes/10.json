{
  "@type": "Recipe",
  "aggregateRating": {
    "@type": "AggregateRating",
    "ratingValue": "4.9",
    "reviewCount": "450"
  },
  "author": {
    "@type": "Person",
    "name": "Sally"
  },
  "cookingMethod": "Baking",
  "cookTime": "PT25M",
  "createdAt": "2020-10-08T05:16:27.181Z",
  "datePublished": "2020-01-01",
  "description": "Even if you've never made homemade bread or worked with yeast before, this homemade artisan bread is for you. Watch the video tutorial above and review the recipe instructions and recipe notes prior to beginning. If you’re new to working with yeast, reference my Baking with Yeast Guide for answers to common yeast FAQs.",
  "image": [
    "https://cdn.sallysbakingaddiction.com/wp-content/uploads/2019/12/homemade-artisan-bread.jpg",
    "https://cdn.sallysbakingaddiction.com/wp-content/uploads/2019/12/homemade-artisan-bread-225x225.jpg",
    "https://cdn.sallysbakingaddiction.com/wp-content/uploads/2019/12/homemade-artisan-bread-260x195.jpg",
    "https://cdn.sallysbakingaddiction.com/wp-content/uploads/2019/12/homemade-artisan-bread-320x180.jpg"
  ],
  "keywords": "bread, loaf",
  "name": "Homemade Artisan Bread (With or Without Dutch Oven)",
  "prepTime": "PT4H",
  "recipeCategory": "Bread",
  "recipeCuisine": "American",
  "recipeIngredient": [
    "3 and 1/4 cups (420g) bread flour (spoon & leveled), plus more for hands and pan",
    "2 teaspoons instant yeast",
    "2 teaspoons coarse salt (see note)",
    "1 and 1/2 cups (360ml) cool water",
    "optional: cornmeal for dusting pan"
  ],
  "recipeInstructions": [
    {
      "@type": "HowToStep",
      "text": "In a large un-greased mixing bowl, whisk the flour, yeast, and salt together. Pour in the cool water and gently mix together with a rubber spatula or wooden spoon. The dough will seem dry and shaggy, but keep working it until all the flour is moistened. If needed, use your hands (as I do in the video tutorial above) to work the dough ingredients together. The dough will be sticky. Shape into a ball in the bowl as best you can."
    },
    {
      "@type": "HowToStep",
      "text": "Cover the dough tightly with plastic wrap or aluminum foil and set on the counter at room temperature (honestly any normal room temperature is fine!). Allow to rise for 2-3 hours. The dough will just about double in size, stick to the sides of the bowl, and have a lot of air bubbles."
    },
    {
      "@type": "HowToStep",
      "text": "You can continue with step 4 immediately, but for absolute best flavor and texture, I strongly recommend letting this risen dough rest in the refrigerator for at least 12 hours and up to 3 days. Place covered dough in the refrigerator for 12 hours – 3 days. I usually let it rest in the refrigerator for about 18 hours. The dough will puff up during this time, but may begin to deflate after 2 days. That’s fine and normal– nothing to worry about."
    },
    {
      "@type": "HowToStep",
      "text": "Lightly dust a large nonstick baking sheet (with or without rims and make sure it’s nonstick) with flour and/or cornmeal. Turn the cold dough out onto a floured work surface. Using a sharp knife or bench scraper, cut dough in half. Some air bubbles will deflate as you work with it. Place dough halves on prepared baking sheet. Using floured hands, shape into 2 long loaves about 9×3 inches each (doesn’t have to be exact) about 3 inches apart. Loosely cover and allow to rest for 45 minutes. You will bake the dough on this prepared baking sheet. See recipe note if you want to use a pizza stone."
    },
    {
      "@type": "HowToStep",
      "text": "During this 45 minutes, preheat the oven to 475°F (246°C)."
    },
    {
      "@type": "HowToStep",
      "text": "When ready to bake, using a very sharp knife or bread lame (some even use kitchen shears), score the bread loaves with 3 slashes, about 1/2 inch deep. (“Score” = shallow cut.) If the shaped loaves flattened out during the 45 minutes, use floured hands to narrow them out along the sides again."
    },
    {
      "@type": "HowToStep",
      "text": "Optional for a slightly crispier crust: After the oven is preheated and bread is scored, place a shallow metal or cast iron baking pan or skillet (I usually use a metal 9×13 baking pan) on the bottom oven rack. Carefully and quickly pour 3-4 cups of boiling water into it. Place the scored dough/baking pan on a higher rack and quickly shut the oven, trapping the steam inside. The steam helps create a crispier crust."
    },
    {
      "@type": "HowToStep",
      "text": "Place the shaped and scored dough (on the flour/cornmeal dusted pan) in the preheated oven on the center rack. Bake for 20-25 minutes or until the crust is golden brown. Gently tap the loaves– if they sound hollow, the bread is done."
    },
    {
      "@type": "HowToStep",
      "text": "Remove the bread from the oven and allow to cool for at least 5 minutes before slicing and serving. Store leftovers loosely covered at room temperature for up to 5 days or in the refrigerator for up to 10 days."
    }
  ],
  "recipeYield": "2",
  "sameAs": ["https://sallysbakingaddiction.com/homemade-artisan-bread/"],
  "suitableForDiet": "https://schema.org/VeganDiet",
  "totalTime": "PT4H25M",
  "updatedAt": "2020-10-08T05:16:27.181Z",
  "url": "https://sallysbakingaddiction.com/homemade-artisan-bread/",
  "video": {
    "@context": "http://schema.org",
    "@type": "VideoObject",
    "contentUrl": "https://www.youtube.com/watch?v=SgXjh5BJ0-M",
    "description": "Even if you’ve never made homemade bread or worked with yeast before, this homemade artisan bread is for you. Watch the video tutorial above and review the recipe instructions and recipe notes prior to beginning.\n\nIf you’re new to working with yeast, reference my Baking with Yeast Guide for answers to common yeast FAQs: https://sallysbakingaddiction.com/baking-with-yeast/\n\nIngredients\n3 and 1/4 cups (420g) bread flour (spoon & leveled), plus more for hands and pan\n2 teaspoons instant yeast\n2 teaspoons coarse salt (see note)\n1 and 1/2 cups (360ml) cool water\noptional: cornmeal for dusting pan\n\nGet the full recipe: https://sallysbakingaddiction.com/homemade-artisan-bread/\n\n• Subscribe and turn on notifications to never miss a new video: https://www.youtube.com/channel/UCXSCbnLpg23d7WU5bIOAsBg?sub_confirmation=1\n\n#baking #recipes #artisanbread\n\nWelcome to my kitchen! I’m Sally, a cookbook author, photographer, and blogger. My goal is to give you the confidence and knowledge to cook and bake from scratch while providing quality recipes and plenty of pictures. Grab a cookie, take a seat, and have fun exploring! \n\n• For more of my favorite recipes go to https://sallysbakingaddiction.com/ \n\n• Follow me\nInstagram: https://www.instagram.com/sallysbakeblog\nFacebook: https://www.facebook.com/sallysbakingaddiction/\nTwitter: https://twitter.com/SallysBakeBlog\nPinterest: https://twitter.com/SallysBakeBlog",
    "duration": "PT4M",
    "embedUrl": "https://www.youtube.com/embed/SgXjh5BJ0-M?feature=oembed",
    "name": "Homemade Artisan Bread (With or Without Dutch Oven) | Sally's Baking Addiction",
    "thumbnailUrl": [
      "https://content.jwplatform.com/thumbs/yXQwx8l1-720.jpg",
      "https://i.ytimg.com/vi/SgXjh5BJ0-M/hqdefault.jpg"
    ],
    "uploadDate": "2020-05-30"
  }
}
