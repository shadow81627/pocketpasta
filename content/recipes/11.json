{
  "@type": "Recipe",
  "aggregateRating": {
    "@type": "AggregateRating",
    "ratingCount": "47",
    "ratingValue": "5"
  },
  "author": {
    "@type": "Person",
    "name": "Kate Hackworthy | Veggie Desserts"
  },
  "cookTime": "PT20M",
  "createdAt": "2020-10-08T05:16:54.247Z",
  "datePublished": "2018-11-18T15:00:26+00:00",
  "description": "These vegan gluten-free Spanish beans are packed with flavour, but are so easy to make in less than 20 minutes with just 7 ingredients! Perfect served with rice or fresh bread and only 125 calories per serving.",
  "image": [
    "https://veggiedesserts.com/wp-content/uploads/2016/06/SpanishWhiteBeans.jpg",
    "https://veggiedesserts.com/wp-content/uploads/2016/06/SpanishWhiteBeans-500x500.jpg",
    "https://veggiedesserts.com/wp-content/uploads/2016/06/SpanishWhiteBeans-500x375.jpg",
    "https://veggiedesserts.com/wp-content/uploads/2016/06/SpanishWhiteBeans-480x270.jpg"
  ],
  "keywords": "spanish beans, spanish recipe, tapas, vegan gluten free",
  "name": "Spanish Beans with Tomatoes",
  "nutrition": {
    "@type": "NutritionInformation",
    "calories": "125 kcal",
    "carbohydrateContent": "19 g",
    "fatContent": "2 g",
    "fiberContent": "6 g",
    "proteinContent": "6 g",
    "servingSize": "1 g",
    "sodiumContent": "2 mg",
    "sugarContent": "3 g"
  },
  "prepTime": "PT5M",
  "recipeCategory": ["Main Meals"],
  "recipeCuisine": "Spanish",
  "recipeIngredient": [
    "1 tablespoon extra virgin olive oil",
    "1 onion (diced)",
    "2 garlic cloves (finely chopped)",
    "1 teaspoon sweet smoked paprika (pimenton dulce - or hot paprika if you like it spicy)",
    "2 bay leaves",
    "2 cans butter beans (400ml/14oz (lima beans), drained and rinsed)",
    "2 cans plum tomatoes (400ml/14oz )",
    "Sea salt and black pepper",
    "2 large handfuls spinach (roughly chopped)"
  ],
  "recipeInstructions": [
    {
      "@type": "HowToStep",
      "text": "In a large saucepan, heat the oil over a medium heat. Add the onions and fry, stirring continuously, for 3-5 minutes until translucent but not browned."
    },
    {
      "@type": "HowToStep",
      "text": "Add the garlic, paprika and bay leaves and fry for a further minute."
    },
    {
      "@type": "HowToStep",
      "text": "Add the beans and tomatoes. Stir and break up the tomatoes a bit, then season with salt and pepper. Stir well and reduce the heat to low. Simmer for 15 minutes, checking regularly so it doesn’t stick to the bottom of the pan."
    },
    {
      "@type": "HowToStep",
      "text": "As it finishes, stir the spinach through to wilt."
    },
    {
      "@type": "HowToStep",
      "text": "Serve hot or cold, as tapas or a side dish, sprinkled with the fresh parsley."
    }
  ],
  "recipeYield": "4",
  "sameAs": ["https://veggiedesserts.com/spanish-beans-tomatoes/"],
  "suitableForDiet": "https://schema.org/VeganDiet",
  "totalTime": "PT25M",
  "updatedAt": "2020-10-08T05:16:54.247Z",
  "video": {
    "@type": "VideoObject",
    "contentUrl": "https://embed.mediavine.com/videos/fyqzt8lidx6wppbpv3gs",
    "description": "These Spanish Beans with tomatoes and smokey sweet spices are easy to make in less than 20 minutes. They’re perfect as tapas, a main meal or a side dish. Vegan and gluten-free. They're a very popular recipe on my site with hundreds of readers loving them.",
    "duration": "PT58S",
    "name": "Spanish Beans",
    "thumbnailUrl": "https://mediavine-res.cloudinary.com/image/upload/s--7DLdAN27--/c_limit,f_auto,fl_lossy,h_1080,q_auto,w_1920/v1515851007/c1i9zamwfqshnvx0ougq.jpg",
    "uploadDate": "2018-01-13T13:39:38+00:00"
  }
}
