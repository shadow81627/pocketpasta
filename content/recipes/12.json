{
  "@type": "Recipe",
  "aggregateRating": {
    "@type": "AggregateRating",
    "ratingCount": "161",
    "ratingValue": "3.85"
  },
  "author": {
    "@type": "Person",
    "name": "Heidi Swanson"
  },
  "cookTime": "PT25M",
  "createdAt": "2020-10-09T02:48:24.108Z",
  "datePublished": "2018-11-15T14:23:18+00:00",
  "description": "A delicious, simple vegetarian split pea soup made from an impossibly short list of ingredients. Seriously, just five!",
  "image": [
    "https://images.101cookbooks.com/vegetarian-split-pea-soup-h2.jpg?w=680&auto=format",
    "https://images.101cookbooks.com/vegetarian-split-pea-soup-2.jpg?w=1000&auto=format",
    "https://images.101cookbooks.com/vegetarian-split-pea-soup-2.jpg?w=800&h=800&fp-z=2&fp-y=.5&fp-x=.5&fit=crop&crop=entropy"
  ],
  "keywords": "split pea soup",
  "name": "Vegetarian Split Pea Soup",
  "nutrition": {
    "@type": "NutritionInformation",
    "calories": "259 kcal",
    "carbohydrateContent": "43 g",
    "fatContent": "3 g",
    "fiberContent": "17 g",
    "proteinContent": "16 g",
    "servingSize": "1 serving",
    "sodiumContent": "409 mg",
    "sugarContent": "6 g"
  },
  "prepTime": "PT5M",
  "recipeCategory": ["Lunch", "Main Course", "Soup"],
  "recipeCuisine": "American",
  "recipeIngredient": [
    "1 tablespoon extra virgin olive oil",
    "2 large onions, chopped",
    "1 teaspoon fine-grain sea salt",
    "2 cups dried split green peas, picked over and rinsed",
    "5 cups water, plus more to finish",
    "juice of 1 lemon (reserve the zest)",
    "a few pinches of smoked paprika, olive oil, toasted almonds, and/or, scallions"
  ],
  "recipeInstructions": [
    {
      "@type": "HowToStep",
      "text": "Add olive oil to a big pot over med-high heat. Stir in onions and salt and cook until the onions soften, just a minute or two. Add the split peas and water. Bring to a boil, dial down the heat, and simmer for 20 minutes, or until the peas are cooked through (but still a touch al dente)."
    },
    {
      "@type": "HowToStep",
      "text": "Using a large cup or mug ladle half of the soup into a bowl and set aside. Using a hand blender (or regular blender) puree the soup that is still remaining in the pot. Stir the reserved (still chunky) soup back into the puree - you should have a soup that is nicely textured. If you need to thin the soup out with more water (or stock) do so a bit at a time - there are times I need to add up to 3 1/2 cups extra."
    },
    {
      "@type": "HowToStep",
      "text": "Stir in the lemon juice and taste. If the soup needs more salt (likely), add more a bit at a time until the flavor of the soup really pops."
    }
  ],
  "recipeYield": "6",
  "sameAs": ["https://www.101cookbooks.com/vegetarian-split-pea-soup/"],
  "suitableForDiet": "https://schema.org/VeganDiet",
  "totalTime": "PT30M",
  "updatedAt": "2020-10-09T04:10:48.766Z"
}
