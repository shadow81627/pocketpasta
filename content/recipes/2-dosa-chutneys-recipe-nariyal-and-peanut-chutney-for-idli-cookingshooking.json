{
  "@type": "Recipe",
  "author": "Yaman Agarwal",
  "cookTime": "PT10M",
  "createdAt": "2020-10-11T23:19:05.204Z",
  "description": "In this video, we will make 2 types of Dosa and Idli Chutneys, these chutneys or pachadis are very famous as accompaniments for South Indian Delicacies like Dosa, Idly, Vada, Pesarattu, Vada, Bhajji etc. \n\nWatch my Sambar Recipe here - https://goo.gl/544aVi\nWatch my Rava Idli here - https://goo.gl/oiJgH8\n\nIngredients:\nCoconut Chutney:\nNariyal - 1 (medium size, peeled & diced)\nPuffed Chana Dal - ½ cup\nCurry Leaves - 1 sprig\nGreen Chili - 2\nGinger - 1/2 inch\nSalt - as per taste\nCumin - 1 tsp\nYogurt - 2 tbsp\nWater - 2 tbsp\nCoriander leaves - handful \n\nFor Takda\nOil - 1 tbsp\nMustard seeds/sarso - 1 tsp\nUrad dal - ½ tsp\nHing/Asafoetida - 1 pinch\nCurry Leaf - 4-5\nTurmeric - ¼ tsp\nWhole red chili - 2\n--\nPeanut Chutney:\nPeanut / Groundnut - ½ cup\nCoconut - 1 tbsp\nSalt -  as per taste\nWhole Red Chili - 6\nGarlic cloves - 5\nCumin seeds - ½ tsp\nTamarind pulpP. - 2 tbsp\nWater - ½ cup\n\nFor tadka\nOil - 1 tbsp\nMustard seeds/Sarso - 1 tsp\nUrad dal - ½ tsp\nWhole red chili - 2\nCurry leaves - 4-5\nTurmeric - ¼ tsp\nJaggery - 1 tsp\n\n\nWebsite: http://www.cookingshooking.com | http://www.cookingshooking.in\nFb: http://www.fb.com/cookingshooking\nBlog: http://www.cooknshook.blogspot.com",
  "image": "https://storage.googleapis.com/cscom-2019.appspot.com/uploads/2018/08/Dosa-Chutneys-1-1.jpg",
  "keywords": "accompaniment, coconut chutney recipe, idli dosa chutney, peanut chutney recipe, south indian coconut chutney",
  "name": "2 Dosa Chutneys Recipe – Nariyal and Peanut Chutney for Idli – CookingShooking",
  "prepTime": "PT25M",
  "recipeIngredient": [
    "1 medium peeled and diced Coconut",
    "1/2 cup or any one roasted chana / cashewnut / peanut Puffed Chana Dal",
    "1 sprig Curry Leaves",
    "2 small Green Chili",
    "1/2 inch Ginger",
    "To taste salt",
    "1 tsp Cumin",
    "2 tbsp or lemon juice yogurt",
    "2 tbsp Water",
    "Handful Coriander"
  ],
  "recipeInstructions": [
    {
      "@type": "HowToStep",
      "image": "https://cookingshooking.com/wp-content/uploads/2018/08/Step-1-24-300x161.jpg",
      "text": "In a mixer grinder jar, combine coconut, curry leaves, cumin, ginger, green chili and salt."
    },
    {
      "@type": "HowToStep",
      "image": "https://cookingshooking.com/wp-content/uploads/2018/08/Step-2-22-176x300.jpg",
      "text": "Next add puffed chana dal, water and coriander leaves. Blend everything until you get a paste like consistency. Add water if required."
    },
    {
      "@type": "HowToStep",
      "image": "https://cookingshooking.com/wp-content/uploads/2018/08/Step-3-26-263x300.jpg",
      "text": "Add yogurt and give it a good stir, and transfer the coconut chutney in a bowl."
    },
    {
      "@type": "HowToStep",
      "image": "https://cookingshooking.com/wp-content/uploads/2018/08/Step-4-24-300x165.jpg",
      "text": "Heat a tadka pan or a small pan over medium flame; add oil and then the mustard seeds, urad dal and hing. Allow it to crackle."
    },
    {
      "@type": "HowToStep",
      "image": "https://cookingshooking.com/wp-content/uploads/2018/08/Step-5-26-300x167.jpg",
      "text": "Add the curry leaves, turmeric powder and finally the whole red chili. Allow everything to crackle."
    },
    {
      "@type": "HowToStep",
      "image": "https://cookingshooking.com/wp-content/uploads/2018/08/Step-6-21-229x300.jpg",
      "text": "Add the tadka to the coconut chutney, and give it a light mix. Your coconut chutney is ready."
    }
  ],
  "sameAs": [
    "https://cscom-2019.web.app/2-dosa-chutneys-recipe-nariyal-and-peanut-chutney-for-idli-cookingshooking/"
  ],
  "serves": null,
  "totalTime": "PT35M",
  "updated": 1581362774000,
  "updatedAt": "2020-10-11T23:19:05.204Z",
  "video": {
    "@type": "VideoObject",
    "description": "In this video, we will make 2 types of Dosa and Idli Chutneys, these chutneys or pachadis are very famous as accompaniments for South Indian Delicacies like Dosa, Idly, Vada, Pesarattu, Vada, Bhajji etc. \n\nWatch my Sambar Recipe here - https://goo.gl/544aVi\nWatch my Rava Idli here - https://goo.gl/oiJgH8\n\nIngredients:\nCoconut Chutney:\nNariyal - 1 (medium size, peeled & diced)\nPuffed Chana Dal - ½ cup\nCurry Leaves - 1 sprig\nGreen Chili - 2\nGinger - 1/2 inch\nSalt - as per taste\nCumin - 1 tsp\nYogurt - 2 tbsp\nWater - 2 tbsp\nCoriander leaves - handful \n\nFor Takda\nOil - 1 tbsp\nMustard seeds/sarso - 1 tsp\nUrad dal - ½ tsp\nHing/Asafoetida - 1 pinch\nCurry Leaf - 4-5\nTurmeric - ¼ tsp\nWhole red chili - 2\n--\nPeanut Chutney:\nPeanut / Groundnut - ½ cup\nCoconut - 1 tbsp\nSalt -  as per taste\nWhole Red Chili - 6\nGarlic cloves - 5\nCumin seeds - ½ tsp\nTamarind pulpP. - 2 tbsp\nWater - ½ cup\n\nFor tadka\nOil - 1 tbsp\nMustard seeds/Sarso - 1 tsp\nUrad dal - ½ tsp\nWhole red chili - 2\nCurry leaves - 4-5\nTurmeric - ¼ tsp\nJaggery - 1 tsp\n\n\nWebsite: http://www.cookingshooking.com | http://www.cookingshooking.in\nFb: http://www.fb.com/cookingshooking\nBlog: http://www.cooknshook.blogspot.com",
    "duration": "PT325S",
    "embedUrl": "https://www.youtube.com/embed/yW9hvteapUk",
    "interactionCount": "423319",
    "name": "2 Dosa Chutneys Recipe - Nariyal and Peanut Chutney for Idli - CookingShooking",
    "sameAs": ["https://www.youtube.com/watch/yW9hvteapUk?autoplay=1"],
    "thumbnailUrl": ["https://i.ytimg.com/vi/yW9hvteapUk/maxresdefault.jpg"],
    "uploadDate": "2018-02-02"
  }
}
