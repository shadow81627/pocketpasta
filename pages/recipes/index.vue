<template>
  <div class="container">
    <list v-bind="{ heading: $t('recipes.heading'), list, layout: 'list' }" />
  </div>
</template>

<script>
import List from '@/components/List/List';
export default {
  components: {
    List,
  },
  data: () => ({ list: [] }),
  head() {
    return {
      meta: [
        {
          hid: 'description',
          name: 'description',
          content: 'Easy pasta recipes made with simple ingredients.',
        },
      ],
      link: [
        {
          hid: 'canonical',
          rel: 'canonical',
          href: 'https://pocketpasta.com/recipes',
        },
      ],
    };
  },
  async asyncData(context) {
    const list = await context.app.$firebase.firestore
      .collection('recipes')
      .get()
      .then(function(querySnapshot) {
        return querySnapshot.docs.map((doc) => doc.data());
      });

    // Thanks for the array merge function
    // https://codeburst.io/javascript-array-distinct-5edc93501dc4
    list.push(context.store.state.recipes);
    const result = [];
    const map = new Map();
    for (const item of list) {
      if (!map.has(item.id)) {
        map.set(item.id, true); // set any value to Map
        result.push(item);
      }
    }

    return { list: result };
  },
};
</script>
