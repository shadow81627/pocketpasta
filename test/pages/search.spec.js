import recipes from '@/pages/search.vue';
import { shallowMount, createLocalVue, RouterLinkStub } from '@vue/test-utils';
import VueRouter from 'vue-router';
import Vuetify from 'vuetify';
import BootstrapVuePlugin from 'bootstrap-vue';
import Vue from 'vue';

const localVue = createLocalVue();

localVue.use(VueRouter);
Vue.use(Vuetify);
localVue.use(BootstrapVuePlugin);

const router = new VueRouter();
let vuetify;

const factory = () =>
  shallowMount(recipes, {
    vuetify,
    mocks: {
      $fetchState: {},
    },
    router,
    stubs: {
      NuxtLink: RouterLinkStub,
    },
    localVue,
  });

describe('recipes', () => {
  beforeEach(() => {
    vuetify = new Vuetify();
  });

  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.vm).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });

  // test('head', () => {
  //   expect(recipes.head()).toBeTruthy();
  // });
});
