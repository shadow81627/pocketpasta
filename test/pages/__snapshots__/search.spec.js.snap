// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`recipes renders properly 1`] = `
"<div class=\\"container\\">
  <list-stub collection=\\"\\" heading=\\"Search\\" layout=\\"list\\" deep=\\"true\\" fetchonserver=\\"true\\" defaultlimit=\\"-1\\"></list-stub>
</div>"
`;
